run_integration_test.go

```diff
@@ -94,6 +95,91 @@ func TestRunsList(t *testing.T) {
 	})
 }
 
+func TestRunsListQueryParams(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	orgTest, orgTestCleanup := createOrganization(t, client)
+	defer orgTestCleanup()
+
+	wTest, _ := createWorkspace(t, client, orgTest)
+	rTest1, _ := createPlannedRun(t, client, wTest)
+	rTest2, _ := createRun(t, client, wTest)
+
+	currentUser, _ := client.Users.ReadCurrent(ctx)
+	userSearch := currentUser.Username
+
+	t.Run("with status query parameter", func(t *testing.T) {
+		skipIfBeta(t)
+		rl, err := client.Runs.List(ctx, wTest.ID, &RunListOptions{
+			Include: []RunIncludeOpt{RunWorkspace},
+			Status:  strings.Join([]string{string(RunPending)}, ","),
```

Comment:
Since we're using only one value in tests, could we not use simplified version => `string(RunPending)` to reduce clutter?